docs/configuration.rst

```diff
@@ -145,6 +145,15 @@ Defaults to ``True``
 
   this only work when PIPELINE_ENABLED is False.
 
+``SHOW_ERRORS_INLINE``
+......................
+
+``True`` if errors compiling CSS/JavaScript files should be shown inline at
+the top of the browser window, or ``False`` if they should trigger exceptions
+(the older behavior).
+
+Defaults to ``True``.
```

Comment:
This should also note that this only applies when running the compilers from the `{% stylesheet %}` and `{% javascript %}` templatetags, and does not affect anything compiled during `collectstatic`.